app/src/server/api/external/v1Api.router.ts

```diff
@@ -334,6 +347,135 @@ export const v1ApiRouter = createOpenApiRouter({
 
       return { status: "ok" };
     }),
+  updateLogTags: openApiProtectedProc
```

Comment:
It's your call since you've done the local testing, but personally this is the kind of method where I'd probably feel much more comfortable writing some tests to make sure all the edge cases around updating/deleting/adding are properly covered, especially since we might easily update the behavior of `parseTags` or `constructLoggedCallFiltersQuery` in the future for something frontend-related and forget it applies here as well.

Reply:
Fair, they're pretty good. The main ones that come to mind are (1) one that doesn't update *any rows* and (2) one that updates 2+ rows (but explicitly doesn't update a third row that matches some but not all of the filters). I'd also inspect the non-updated rows to make sure they aren't actually updated -- I realize that isn't really a thing we can do easily from the SDK side though.

Reply:
I added those tests to the sdk.